legal-api/report-templates/template-parts/bc-address-change/addresses.html

```diff
@@ -1,39 +1,39 @@
 <div class="address-container">
-  {% if bcAddressChange.offices.registeredOffice is defined %}
+  {% if  registeredOfficeAddress is defined %}
   <div class="section-title">
     Registered Office Information
   </div>
   <div class="section-data">
     <table class="section-data-table" role="presentation">
       <tr>
         <td class="delivery-address">
-          <div class="section-sub-title">Delivery Address</div>
-          <div>{{ bcAddressChange.offices.registeredOffice.deliveryAddress.streetAddress }}</div>
-          <div>{{ bcAddressChange.offices.registeredOffice.deliveryAddress.streetAddressAdditional }}</div>
-          <div>{{ bcAddressChange.offices.registeredOffice.deliveryAddress.addressCity }}
-            {{ bcAddressChange.offices.registeredOffice.deliveryAddress.addressRegion }}
-            &nbsp;{{ bcAddressChange.offices.registeredOffice.deliveryAddress.postalCode }}
+          <div class="section-sub-title">Delivery Address  {% if 'addressChanged' in registeredOfficeAddress.deliveryAddress.actions %}{{ labelNew() }}{% endif %}</div>
+          <div> {{ registeredOfficeAddress.deliveryAddress.streetAddress }}</div>
```

Comment:
This is probably a non-breaking space anyway, but why did you add it for all address lines?

Reply:
I did a search and replace and I think extra space got added in the beginning on the replace text. Will remove that